LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -221,21 +237,35 @@ public void CreateClassrooms(int classroomCount)
         {
             if (ModuleIdList.Count != 0)
             {
+                ReportHeader(actionName: "Classroom generation");
+
                 for (int i = 0; i < classroomCount; i++)
                 {
                     Guid moduleId = ModuleIdList[AttributeGenerator.random.Next(0, ModuleIdList.Count)];
                     Module module;
                     ModuleDictionary.TryGetValue(moduleId, out module);
                     Classroom classroom = ObjectGenerator.GenerateClassroom(module);
                     ClassroomDictionary.Add(classroom.Id, classroom);
+
+                    ReportItem(itemName: classroom.ToString(), actionName: "Generate", itemIndex: ++i);
                 }
+
+                ReportSummary(summary: "Classroom", count: classroomCount);
```

Comment:
Move ReportSummary and ReportFooter out of the **if** block to the end of the method.

Reply:
Done